app/src/main/java/com/nervousfish/nervousfish/modules/cryptography/KeyGeneratorAdapter.java

```diff
@@ -20,6 +20,8 @@
  */
 public final class KeyGeneratorAdapter implements IKeyGenerator {
     private static final Logger LOGGER = LoggerFactory.getLogger("KeyGeneratorAdapter");
+    private static final String KEY_ALGORITHM = "RSA";
+    private static final int KEY_SIZE = 2048;
```

Comment:
Should be something like `RSA_KEY_SIZE` since this class is not meant solely for RSA key generation.